net-im/skypeforlinux/skypeforlinux-5.4.0.1.ebuild

```diff
@@ -7,7 +7,7 @@ MULTILIB_COMPAT=( abi_x86_64 )
 
 inherit pax-utils rpm multilib-build
 
-DESCRIPTION="P2P Internet Telephony (VoiceIP) client"
+DESCRIPTION="Internet Telephony (VoIP) and messaging client, with support for audio and video"
```

Comment:
could you make the capitalization consistent, please? Does "Telephony" not include audio? 
I would suggest to find a better, clearer DESC. (note the 80 char limit)